bindings/python/src/pre_tokenizers.rs

```diff
@@ -489,17 +489,40 @@ impl PyMetaspace {
         setter!(self_, Metaspace, add_prefix_space, add_prefix_space);
     }
 
+    #[getter]
+    fn get_prepend_scheme(self_: PyRef<Self>) -> String {
+        getter!(self_, Metaspace, prepend_scheme.to_string())
+    }
+
+    #[setter]
+    fn set_prepend_scheme(self_: PyRef<Self>, prepend_scheme: String) {
+        setter!(self_, Metaspace, @set_prepend_scheme, prepend_scheme);
+    }
+
     #[new]
     #[pyo3(signature = (replacement = PyChar('▁'), add_prefix_space = true, **_kwargs), text_signature = "(self, replacement=\"_\", add_prefix_space=True)")]
     fn new(
         replacement: PyChar,
         add_prefix_space: bool,
         _kwargs: Option<&PyDict>,
     ) -> (Self, PyPreTokenizer) {
-        (
-            PyMetaspace {},
-            Metaspace::new(replacement.0, add_prefix_space).into(),
-        )
+        // Create a new Metaspace instance
+        let mut new_instance: Metaspace = Metaspace::new(replacement.0, add_prefix_space);
```

Comment:
I don't use the `new_with_prepend_type` because otherwise I need to implement the PyPrependScheme, which I'd rather avoid. My question here @Narsil is whether or not I should just allow string prepend scheme in the `new_with_prepend` using the `set_prepend_scheme` internally?

Reply:
string for enums seems the easiest way.